pageserver/src/tenant/timeline.rs

```diff
@@ -450,7 +450,7 @@ pub(crate) struct GcInfo {
     /// Currently, this includes all points where child branches have
     /// been forked off from. In the future, could also include
     /// explicit user-defined snapshot points.
-    pub(crate) retain_lsns: Vec<(Lsn, TimelineId)>,
+    pub(crate) retain_lsns: Vec<(Lsn, TimelineId, bool)>,
```

Comment:
I'm guilty of doing this myself sometimes, but a type would be much more readable here.
Something like:
```
enum BranchPoint {
  Active(...),
  Archived(...)
}
```

Reply:
Done